.github/workflows/pr-tests-stack.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         if: steps.changes.outputs.stack == 'true'
         run: |
           python -m pip install --upgrade pip
-          pip install uv==0.4.1 tox==4.18.0 tox-uv==1.11.2
+          pip install uv==0.4.16 tox==4.20.0 tox-uv==1.13.0
```

Comment:
let's not bump any version for now :) Minimize any variables that may cause problems

Reply:
thanks Yash. Reverted it back